src/cdk/a11y/_a11y.import.scss

```diff
@@ -0,0 +1,2 @@
+@forward 'a11y' hide a11y, high-contrast;
```

Comment:
By introducing these `.import` files, Sass will automatically use them for `@import` statements without needing to modify the import path, right?